tensorboard/plugins/debugger/tensor_helper_test.py

```diff
@@ -101,6 +101,42 @@ def testImagePngMappingWorksForArrayWithInfAndNaN(self):
     self.assertAllClose([255, 255, 255], decoded_x[1, 1, :])  # 3.3.
     self.assertAllClose(tensor_helper.NAN_RGB, decoded_x[1, 2, :])  # nan.
 
+  def testArrayViewSlicingDownNumericTensorToOneElement(self):
+    x = np.array([[1.1, 2.2, np.inf], [-np.inf, 3.3, np.nan]], dtype=np.float32)
+    dtype, shape, data = tensor_helper.array_view(x, slicing='[0,0]')
+    self.assertEqual('float32', dtype)
+    self.assertEqual(tuple(), shape)
+    self.assertTrue(np.allclose(1.1, data))
+
+  def testArrayViewSlicingStringTensorToNonScalarSubArray(self):
+    # Construt a numpy array that corresponds to a TensorFlow string tensor
+    # value.
+    x = np.array([['foo', 'bar', 'qux'], ['baz', 'corge', 'grault']],
+                 dtype=np.object)
+    dtype, shape, data = tensor_helper.array_view(x, slicing='[:2, :2]')
+    self.assertEqual('string', dtype)
+    self.assertEqual((2, 2), shape)
+    self.assertEqual([['foo', 'bar'], ['baz', 'corge']], data)
+
+  def testArrayViewSlicingStringTensorToScalar(self):
+    # Construt a numpy array that corresponds to a TensorFlow string tensor
+    # value.
+    x = np.array([['foo', 'bar', 'qux'], ['baz', 'corge', 'grault']],
+                 dtype=np.object)
+    dtype, shape, data = tensor_helper.array_view(x, slicing='[1, 1]')
+    self.assertEqual('string', dtype)
+    self.assertEqual((1, 1), shape)
+    self.assertEqual([['corge']], data)
+
+  def testArrayViewOnScalarString(self):
+    # Construt a numpy array that corresponds to a TensorFlow string tensor
```

Comment:
Maybe s/array/scalar to make this comment more clear?

Reply:
Done.